Streams.subproj/MPWURLFetchStream.m

```diff
@@ -186,7 +186,14 @@ -(NSURL*)resolve:(NSURL*)theURL
 
```

Comment:
I don't know if we used this, but 
```-(void)writeDictionary:(NSDictionary *)aDictionary
 {
     [self writeObject:[self serializeDictionary:aDictionary]];
 }
```

Also possibly needs to be called on the target thread.